zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -275,6 +280,51 @@ def _do_cleanup(self, metadataPath):
+            return {'error': "listdir failed for %s: %s" % (metadataDir, e)}
+
+        for fname in names:
+            if not meta_name_re.match(fname):
+                continue
+            fpath = os.path.join(metadataDir, fname)
+            if not os.path.isfile(fpath):
+                continue
+            try:
+                os.remove(fpath)
+                removed_any = True
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)